app/src/main/java/org/mozilla/fenix/search/SearchDialogFragment.kt

```diff
@@ -102,7 +104,8 @@ class SearchDialogFragment : AppCompatDialogFragment(), UserInteractionHandler {
         // Let's reset back to the default behavior after we're done searching
         // This will be addressed on https://github.com/mozilla-mobile/fenix/issues/17805
         @Suppress("DEPRECATION")
-        requireActivity().window.setSoftInputMode(WindowManager.LayoutParams.SOFT_INPUT_ADJUST_RESIZE)
+        if (VERSION.SDK_INT < VERSION_CODES.R)
+            requireActivity().window.setSoftInputMode(WindowManager.LayoutParams.SOFT_INPUT_ADJUST_RESIZE)
```

Comment:
Same question here

Reply:
Nope. The inset listener set up in the activities will handle that case.